crates/viewer/re_space_view_spatial/src/visualizers/utilities/proc_mesh_vis.rs

```diff
@@ -0,0 +1,316 @@
+use re_entity_db::InstancePathHash;
+use re_log_types::Instance;
+use re_renderer::renderer::MeshInstance;
+use re_renderer::{LineDrawableBuilder, PickingLayerInstanceId, RenderContext};
+use re_types::components::{self, FillMode};
+use re_viewer_context::{
+    QueryContext, SpaceViewSystemExecutionError, TypedComponentFallbackProvider, ViewQuery,
+};
+
+use crate::contexts::SpatialSceneEntityContext;
+use crate::instance_hash_conversions::picking_layer_id_from_instance_path_hash;
+use crate::proc_mesh::{self, ProcMeshKey};
+use crate::visualizers::{
+    process_annotation_and_keypoint_slices, process_color_slice, process_labels_3d,
+    process_radius_slice,
+    utilities::{entity_iterator::clamped_or_nothing, LabeledBatch},
+    SpatialViewVisualizerData, SIZE_BOOST_IN_POINTS_FOR_LINE_OUTLINES,
+};
+
+#[cfg(doc)]
+use re_viewer_context::VisualizerSystem;
+
+/// To be used within the scope of a single [`VisualizerSystem::execute()`] call
+/// when the visualizer wishes to draw batches of [`ProcMeshKey`] meshes.
+pub(crate) struct ProcMeshVisBuffer<'ctx, Fb> {
```

Comment:
`VisBuffer` sounds too much like `VisibilityBuffer` to me (which makes me think of occlusion culling).
What do you think of `ProcMeshDrawableBuilder`?

Reply:
Sounds great! Done!